domain/src/main/java/org/oppia/domain/topic/StoryProgressController.kt

```diff
@@ -69,10 +77,38 @@ class StoryProgressController @Inject constructor() {
     return mapOf(
       TEST_STORY_ID_0 to createStoryProgress0(),
       TEST_STORY_ID_1 to createStoryProgress1(),
-      TEST_STORY_ID_2 to createStoryProgress2()
+      TEST_STORY_ID_2 to createStoryProgress2(),
+      FRACTIONS_STORY_ID_0 to createStoryProgressForJsonStory("fractions_stories.json", /* index= */ 0)
     )
   }
 
+  private fun createStoryProgressForJsonStory(fileName: String, index: Int): TrackedStoryProgress {
```

Comment:
We don't actually want to retrieve story progress from the static JSON files. Progress needs to be dynamic (as part of #21) in order for progress to be reflected.

Reply:
So should I remove this one in this PR and the other dummy progress data in the follow up where I will be removing dummy data?

Reply:
Sorry wait, isn't this the initial story progress that is created?